config/service.py

```diff
@@ -286,6 +286,18 @@
 
 PRIMARY_NAVIGATION = [
     {
+        "label": "About",
+        "url": {
+            "url": "https://monitor.jisc.ac.uk/uk/about",
+        },
+        "visibility": {
+            "auth": True,
+            "anonymous": False
```

Comment:
As added in pivotal tracker: The answer is Yes. However removing or changing the auth property to False doesn't produce any results. Will need further analysis.